examples/image-to-text/README.md

```diff
@@ -131,6 +132,15 @@ python3 run_pipeline.py \
     --sdp_on_bf16
 ```
 
+To run THUDM/glm-4v-9b inference, use the following command (Note that we need a env variable "GLM=4v" to distinguish between glm4v and chatglm, this is because these models are all customized models and share the same model_type named "chatglm"):
```

Comment:
```suggestion
To run inference with THUDM/glm-4v-9b, use the following command (Note that you need to set the environment variable `GLM=4v` to distinguish between glm4v and chatglm, as these models are customized and share the same model type named "chatglm"):
```

Reply:
@mengker33 GLM-4V requires the Tiktoken library to be installed, please update the requirements.

Reply:
you need to add tiktoken and blobfile packages

Reply:
Thanks for your comment :) I added tiktoken in image-to-text/requirements, but I didn't find GLM-4V's dependency for blobfile.

Reply:
Please look at https://github.com/huggingface/transformers/blob/53fad641cfdb5105e2470bcf3ef17ea8e25cc300/src/transformers/utils/import_utils.py#L1202

```sh
def is_tiktoken_available():
    return _tiktoken_available and _blobfile_available
```